ReSharper.FSharp/src/FSharp/FSharp.Common/src/ProjectModel/FSharpLanguageVersion.fs

```diff
@@ -31,6 +31,9 @@ type FSharpLanguageLevel =
     /// Nested record field copy and update/shorthand lambda
     | FSharp80 = 80
 
+    /// Includes fixes for shorthand lambda
+    | FSharp8Patched = 81
```

Comment:
What does `patched` mean in terms of the language level? It's seems too vague and can be about any kind of patch in the language.

Reply:
I'd probably vote for having our own non-existent version like `FSharp81` with the comment like `Shorthand lambda fixes`.